src/main/java/com/rcv/ContestConfig.java

```diff
@@ -36,7 +36,6 @@
 
 class ContestConfig {
 
-  // TODO: if any booleans are unspecified in config file, they default to false no matter what
```

Comment:
I think we do need to keep some kind of warning / reminder to us or any future coders about this... a TODO probably wasn't the best way to do it, but it's important.